server/hub.go

```diff
@@ -230,7 +230,7 @@ func (h *Hub) run() {
 
 				log.Printf("Hub. Topic[%s] is unknown or offline", msg.RcptTo)
 
-				msg.sess.queueOut(NoErrAccepted(msg.Id, msg.RcptTo, types.TimeNow()))
+				msg.sess.queueOut(NoErrAcceptedExplicitTs(msg.Id, msg.RcptTo, types.TimeNow(), msg.Timestamp))
```

Comment:
Yes, it's an interesting idea.
I've gone through the calls, and in many cases (especially in `topic.go`) are as follows :
`Name(id, t.original(asUid), now)`
From looking at cases, it appears `t.original(asUid)` should always be the same as `msg.Original` unless I overlooked something. What was the thought/intent behind using `t.original(asUid)` instead of `msg.Original`? Or was there nothing specific in mind?

Reply:
> should always be the same as msg.Original

Except the case of creating a new group topic when `msg.Original` is `newXXX` or `nchXXX`.

Reply:
> What was the thought/intent behind using t.original(asUid) instead of msg.Original?

1. Legacy
2. The case of `newXXX` or `nchXXX`.

Reply:
The `msg.Original` was introduced very recently with the cluster refactoring.